armada-ths/ais
# Event image upload

Implemented image uploading for event image.

This image will be shown when editing that event or when applying for that event.

To test this:

* make sure you have a fair, I assume it's 2017 fair
* goto localhost:8000/fairs/2017/events/
* create a new event, fill out required fields and an image of your choice
* save
* either click on the event or click edit to see the picture
